test/index.js

```diff
@@ -1,27 +1,37 @@
-console.time('test')
-
-require('esbuild-register/dist/node').register()
-
-const test = require('baretest')('gretchen')
-const assert = require('assert')
-
-require('./utils.test').default(test, assert)
-require('./handleRetry.test').default(test, assert)
-require('./handleTimeout.test').default(test, assert)
-require('./index.test').default(test, assert)
-require('./merge.test').default(test, assert)
-
-process.on('unhandledRejection', e => {
-  console.error(e)
-  process.exit(1)
-})
+/* eslint-env node */
+console.time("test");
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("esbuild-register/dist/node").register();
+
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const test = require("baretest")("gretchen");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const assert = require("assert");
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("./utils.test").default(test, assert);
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("./handleRetry.test").default(test, assert);
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("./handleTimeout.test").default(test, assert);
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("./index.test").default(test, assert);
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+require("./merge.test").default(test, assert);
```

Comment:
Lots of `eslint-disable-next-line` here since this looks like a Node process that uses `require` instead of ESM.